11/architectures

```diff
@@ -0,0 +1,8 @@
+bashbrew-arch   variants
+amd64    alpine,stretch,stretch-slim
+arm32v6  alpine
+arm32v7  stretch,stretch-slim
+arm64v8  alpine,stretch,stretch-slim
+i386     alpine
+ppc64le  alpine,stretch,stretch-slim
+s390x    alpine,stretch,stretch-slim
```

Comment:
The content of this file should be in the root architecture as it would apply to 11 and all future versions.

Reply:
Unless we get rid of the root one altogether and have each version has it's own list until we implement what you suggested

Reply:
So, this is where things got weird.  What I found in testing was that the root file was somehow being used to determine the list of architecture+variant combinations that it was willing to try, so when I copied this file as-is to the root, all the `jessie` variants went away completely, which is why I've updated the root file here to simply list every possible variant for every supported architecture and opted instead to be explicit per-version which arch+variant combinations are supported.

Reply:
AAAh gotcha! Sounds like we need to streamline!